src/roce_util.py

```diff
@@ -13,8 +13,10 @@ def check_pkt_size(mtu, pkt):
         op = pkt[BTH].opcode
         if RC.first_req_pkt(op) or op == RC.RDMA_READ_RESPONSE_FIRST:
             assert len(pkt[Raw].load) == mtu
+            assert pkt[BTH].padcount == 0
         elif RC.mid_req_pkt(op) or op == RC.RDMA_READ_RESPONSE_MIDDLE:
             assert len(pkt[Raw].load) == mtu
+            assert pkt[BTH].padcount == 0
```

Comment:
Shall we check padcount for write packets on the responser side?

Reply:
This function is called both in SQ and RQ

Reply:
But this function only check read related ops.